opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -587,6 +611,81 @@ namespace Opm {
         m_rootGroupTree->add(currentStep, newTree);
     }
 
+    void Schedule::handleWRFT(DeckKeywordConstPtr keyword, LoggerPtr /*logger*/, size_t currentStep) {
+
+        for (size_t recordNr = 0; recordNr < keyword->size(); recordNr++) {
+            DeckRecordConstPtr record = keyword->getRecord(recordNr);
+
+            const std::string& wellNamePattern = record->getItem("WELL")->getTrimmedString(0);
+            const std::vector<WellPtr>& wells = getWells(wellNamePattern);
+
+            for (auto wellIter=wells.begin(); wellIter != wells.end(); ++wellIter) {
+                WellPtr well = *wellIter;
+                well->setRFT(currentStep, true);
+            }
+        }
+
+        const std::vector<WellPtr>& wells = getWells("*");
+        for (auto wellIter=wells.begin(); wellIter != wells.end(); ++wellIter) {
```

Comment:
This looks a bit weird:

1. Set the RFT flag from now and the rest of the simulation for all wells matching the name.
2. Set all the wells to output an RFT when they first open